tests/Issues/IssueChangeAndIfAndDateTimeAndDeMorgan/config/configured_rule.php

```diff
@@ -10,5 +10,5 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->rule(SimplifyDeMorganBinaryRector::class);
     $rectorConfig->rule(ChangeAndIfToEarlyReturnRector::class);
-    $rectorConfig->rule(DateTimeToDateTimeInterfaceRector::class);
+//    $rectorConfig->rule(DateTimeToDateTimeInterfaceRector::class);
```

Comment:
this handled at https://github.com/rectorphp/rector-src/pull/2274 for Scope not avaiable after original node verified as not Node (null)

Reply:
After this PR the scope must be available everywhere. This is just temporary.